bin/schema-registry-run-class

```diff
@@ -63,7 +63,7 @@ fi
 
 # Memory options
 if [ -z "$SCHEMA_REGISTRY_HEAP_OPTS" ]; then
-  SCHEMA_REGISTRY_HEAP_OPTS="-Xmx256M"
+  SCHEMA_REGISTRY_HEAP_OPTS="-Xmx512M"
```

Comment:
This seems quite large for what the schema registry is doing. Did you actually run into limits with the existing setting?